confignode/src/main/java/org/apache/iotdb/confignode/manager/ClusterQuotaManager.java

```diff
@@ -46,10 +49,16 @@ public class ClusterQuotaManager {
 
   private final IManager configManager;
   private final QuotaInfo quotaInfo;
+  private final Map<Integer, Integer> deviceNum;
+  private final Map<Integer, Integer> timeSeriesNum;
```

Comment:
It better uses a concurrent map, it will be updated concurrently during a heartbeat（because the callback thread may run concurrently.）.